Add Salla

jyotimanglani opened the pull request:
![image](https://github.com/simple-icons/simple-icons/assets/104571605/477180e7-a09d-4dc0-8ed0-f9b420436b9e)

**Issue:** closes #10292 

**Similarweb rank:**[228,755](https://www.similarweb.com/website/salla.com/#overview)

PeterShaggyNoble commented:
Thanks for the contribution, @jyotimanglani 👍🏻

I've just now found a [branding site](https://brand.salla.com) for Salla which includes [icons](https://brand.salla.com/#icons-design) and [colours](https://brand.salla.com/#brand-colors). Would you mind double checking the changes in your PR against what's available there before we proceed to review, please?

jyotimanglani commented:
Thanks, @PeterShaggyNoble, I'll review and align my changes with the official branding site for Salla.

jyotimanglani commented:
@PeterShaggyNoble, I've noticed a slight color variation between my logo and the versions on the official branding site. I initially used the hex code #BBF3E5 from the website header, but it seems they often use a darker version with the hex code #004956. Should I modify the hex code to match the darker color (#004956), or is the current one (#BBF3E5) acceptable?

PeterShaggyNoble commented:
I would have initially gone for the darker green when I first saw the request, @jyotimanglani, but the brand guidelines make it pretty clear that the lighter green is actually the primary colour. Of the 3 shades, `#BAF3E6` is closest to what they use on the branding site for the backgrounds of the section heading, so that would get my vote.

jyotimanglani commented:
Thanks, @PeterShaggyNoble! I'll switch to #BAF3E6 as suggested.

jyotimanglani commented:
@PeterShaggyNoble, I've updated the hex code and pushed changes. Could you share the link or steps to regenerate the icon preview for the PR description?

Thanks!

PeterShaggyNoble commented on _data/simple-icons.json:
```diff
@@ -12724,6 +12724,11 @@
+        {
+            "title": "Salla",
+            "hex": "BAF3E6",
+            "source": "https://cdn.salla.network/images/logo/logo-light.svg"
```
```suggestion
            "source": "https://brand.salla.com",
            "guidelines": "https://brand.salla.com"
```

PeterShaggyNoble reviewed (changes requested):
@jyotimanglani, the inner shape of your icon (in red) is [misaligned](https://github.com/simple-icons/simple-icons/assets/15157491/325df697-cb56-4564-8ad9-42b69f70f42d) when compared with [the icon](https://uploads-ssl.webflow.com/63fe4a8108c3c17052878ef9/63fe902f6b9768e45a0aff29_Group%20(1).svg) available in the [icons section](https://brand.salla.com/#icons-design) of the brand guidelines, and I'm also seeing some hairline differences in the outer shape.

We should also update the `source` to the branding site and add a `guidelines` entry with the same URL.

You can generate previews at https://wasm.simpleicons.org/preview/ or https://petershaggynoble.github.io/SI-Sandbox/preview/

jyotimanglani commented:
@PeterShaggyNoble , 
Thanks for your feedback. I've addressed the misalignments in the inner shape and adjusted the outer shape to match the brand guidelines more closely. Additionally, I've updated the source URL to the branding site and added a guidelines entry with the same URL. Please take a look at the changes, and let me know if further adjustments are needed.
Appreciate your guidance!

PeterShaggyNoble reviewed (approved):
Everything looks perfect to me now, @jyotimanglani 👍🏻 Thanks for all your work on this one.
